src/app/likes/likes.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-likes',
+  templateUrl: './likes.component.html',
+  styleUrls: ['./likes.component.css']
+})
+export class LikesComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
```

Comment:
nit: I know this is the result of `ng new`, but typically these should be removed before submission since they are unused, and added back when/if you need them